extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -632,7 +632,7 @@ public String getErrorMessage() {
     private static final String FORMATS = "formats";
     private static final String ADAPTIVE_FORMATS = "adaptiveFormats";
     private static final String HTTPS = "https:";
-    private static final String DEOBFUSCATION_FUNC_NAME = "decrypt";
+    private static final String DEOBFUSCATION_FUNC_NAME = "deobfuscate";
```

Comment:
This doesn't seem okay to me. That name is string matched against the page contents. You shouldn't change it.

Reply:
No, that name is not string-matched anywhere. It is only used as the name of the JS function that is manually built to then call the other functions inside the player's JS.